processor/pom.xml

```diff
@@ -92,6 +92,17 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <MapStruct-Generated-Comment>${project.groupId}:${project.artifactId}:${project.version}</MapStruct-Generated-Comment>
```

Comment:
So: the `@Generated#comments` will look like:
'Implementation-Version: 1.0.0-Beta3, Build-Jdk: 1.8.0_25'